numpy-stubs/__init__.pyi

```diff
@@ -450,6 +451,11 @@ def array(
     subok: bool = ...,
     ndmin: int = ...,
 ) -> ndarray: ...
+def asarray(
+    a: Union[SupportsFloat, Sequence, ndarray],
+    dtype: _DtypeLike = ...,
+    order: str = ...,
```

Comment:
Should this be optional? Looking at https://docs.scipy.org/doc/numpy/reference/generated/numpy.asarray.html